.github/ISSUE_TEMPLATE/Feature_Request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature Request/Idea 🚀
+about: Suggest a new feature or improvement (this does not mean you have to implement it)
+title: "Feature Request"
+labels: Issue-Feature
+assignees: ''
+
+---
+
+# Summary of the new feature/enhancement
+
+<!-- 
+A clear and concise description of what the problem is that the new feature would solve.
+Try formulating it in user story style (if applicable):

```

Comment:
"user story" is not a term I think the general community may understand, formulating is also very "formal".

Perhaps:
Describe why and how a user would use this new functionality (if applicable):

Reply:
> "user story" is not a term I think the general community may understand

To be fair, I'm not sure _I_ know what a "user story" is.